packages/kit/test/test.js

```diff
@@ -86,12 +110,29 @@ function duplicate(test_fn, config) {
 				let response;
 
 				if (start) {
-					response = await context.page.goto(context.base + start);
-					await context.page.evaluate(() => window.start());
+					response = await context.pages.js.goto(context.base + start);
+					await context.pages.js.evaluate(() => window.started);
 				}
 
 				await callback({
 					...context,
+					page: context.pages.js,
+					click: async (selector) => {
```

Comment:
What happens if a test wants to perform a click that doesn't result in a navigation? Would it hang waiting for these events? Should `click` be called something more indicative like `clickAndWaitForNavigation` (or a better-sounding version of that), so people know to use the regular `.click` for other types of clicking?

Reply:
`clicknav`?

Reply:
Yeah I think `clicknav` is probably evocative enough without being unwieldy :+1:

Reply:
If we rename it, I'd call it something shorter like `navClick`